protocols/v2/binary-sv2/no-serde-sv2/codec/src/codec/decodable.rs

```diff
@@ -235,6 +308,11 @@ impl PrimitiveMarker {
         }
     }
 
+    // Decodes a primitive value from a reader stream, returning the corresponding
+    // `DecodablePrimitive`. This is useful when reading data from a file or network socket,
+    // where the data is not immediately available as a slice but must be read incrementally.
+    #[allow(clippy::wrong_self_convention)]
+    #[cfg(not(feature = "no_std"))]
```

Comment:
why are we duplicating these two lines?

Reply:
Nice catch!!! might have come during rebase..